File: solutions/python/darts/2/darts.py
from math import sqrt

SCORES = {
    1: 10,
    5: 5,
    10: 1,
}
def score(x, y):
    distance = sqrt(x**2 + y**2)
    for radius, points in SCORES.items():
        if distance <= radius:
            return points
    return 0

File: solutions/cpp/darts/2/darts.cpp
#include "darts.h"

namespace darts {
    int score(double x, double y) {
        double distance_squared = x * x + y * y; 
        if (distance_squared > 100) {
            return 0;
        } else if (distance_squared > 25) {
            return 1;
        } else if (distance_squared > 1) {
            return 5;
        } else {
            return 10;
        }
    }
} // namespace darts